build/fonts/firasans/firasans-regular-demo/index.html

```diff
@@ -0,0 +1,622 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
This sample html documents need to be ignored.
In the root of this project create a file called `.eleventyignore` and add the paths to the sample pages to it.
e.g, `build/fonts/firasans/firasans-regular-demo/index.html`